ddtrace/appsec/iast/_taint_utils.py

```diff
@@ -21,12 +29,24 @@ def get(self, key, default=None):
 
     def items(self):
         for k, v in super(LazyTaintDict, self).items():
-            if isinstance(v, (str, bytes, bytearray)) and not is_pyobject_tainted(v):
-                v = taint_pyobject(v, Input_info(k, v, 0))
-                super(LazyTaintDict, self).__setitem__(k, v)
+            if v and isinstance(v, (str, bytes, bytearray)) and not is_pyobject_tainted(v):
+                try:
+                    v = taint_pyobject(v, Input_info(k, v, 0))
+                    super(LazyTaintDict, self).__setitem__(k, v)
+                except SystemError:
+                    # TODO: Find the root cause for
+                    # SystemError: NULL object passed to Py_BuildValue
+                    pass
 
             yield (k, v)
 
     def values(self):
         for k, v in self.items():
             yield v
+
+
+def check_tainted_args(args, kwargs, tracer, integration_name, method):
+    if integration_name in DBAPI_INTEGRATIONS and method.__name__ == "execute":
```

Comment:
don't we need to check the lower case of integration_name?

Reply:
I believe they're all lowercase